contracts/scripts/zksync_compile_all

```diff
@@ -24,3 +24,7 @@ compileContract shared/token/ERC677/LinkToken shared "--use 0.8.19"
 compileContract shared/token/ERC677/BurnMintERC677 shared "--use 0.8.19"
 compileContract vendor/multicall/ebd8b64/src/Multicall3 shared "--use 0.8.19"
 compileContract keystone/CapabilitiesRegistry keystone
+
+compileContract vendor/canonical-weth-zk/WETH9 shared "--use 0.8.19"
```

Comment:
canonical-weth-zk should be the repo name for anything in vendor. This looks like we made it up?